be/src/vec/sink/writer/vtablet_writer_v2.cpp

```diff
@@ -651,9 +694,12 @@ Status VTabletWriterV2::_close_wait(bool incremental) {
                                      << print_id(_load_id);
                         return Status::TimedOut("load timed out before close waiting");
                     }
-                    RETURN_IF_ERROR(stream->close_wait(_state, remain_ms));
+                    auto st = stream->close_wait(_state, remain_ms);
+                    if (!st.ok() && status.ok()) {
+                        status = st;
+                    }
                 }
-                return Status::OK();
+                return status;
```

Comment:
Will there be a load_stram leak here? If there is an error, it will be returned.

Reply:
This is inside lambda, see `LoadStreamMap::for_each_st()`.